mods/restore-button-pulse-anim.wh.cpp

```diff
@@ -293,6 +326,16 @@ HMODULE LoadComCtlModule(void)
     ULONG_PTR ulCookie;
     ActivateActCtx(hActCtx, &ulCookie);
     HMODULE hComCtl = LoadLibraryW(L"comctl32.dll");
+    /**
+      * Certain processes will ignore the activation context and load
+      * comctl32.dll 5.82 anyway. If that occurs, just reject it.
+      */
+    VS_FIXEDFILEINFO *pVerInfo = GetModuleVersionInfo(hComCtl, nullptr);
+    if (HIWORD(pVerInfo->dwFileVersionMS) < 6)
```

Comment:
I didn't see comctl32.dll without version info, but IMO i wouldn't hurt to be defensive here just in case.

```suggestion
    if (pVerInfo && HIWORD(pVerInfo->dwFileVersionMS) < 6)
```